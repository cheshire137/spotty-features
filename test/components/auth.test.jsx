import React from 'react'
import renderer from 'react-test-renderer'
import { mount } from 'enzyme'

import Auth from '../../src/components/auth.jsx'

describe('Auth', () => {
  test('matches snapshot', () => {
    const component = <Auth />
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
