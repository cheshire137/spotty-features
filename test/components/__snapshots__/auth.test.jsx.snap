exports[`Auth matches snapshot 1`] = `
<section
  className="hero">
  <div
    className="hero-body">
    <div
      className="container">
      <h2
        className="subtitle">
        Sign in with your Spotify account to see trends about your saved songs.
      </h2>
      <p>
        <a
          className="spotify-button is-primary button is-large"
          href="https://accounts.spotify.com/authorize?response_type=token&redirect_uri=about%3A%2F%2F%2Fauth&client_id=c093721d396d4c3f802e4a65bb653a68&scope=user-library-read%20playlist-modify-public">
          Sign into Spotify
        </a>
      </p>
    </div>
  </div>
</section>
`;
